### PR TITLE
Added email validation method in user service

### PR DESCRIPTION
**Pull request by gariktepanosian**
…s file and changed a css name in general for all html pages.

### CODE REVIEW

File: src/main/java/com/epam/edumanagementsystem/util/service/UserService.java [hunk 1]
```diff
@@ -13,4 +13,6 @@ public interface UserService {
     List<User> findAll();
 
     User findByEmail(String email);
+
+    Boolean checkDuplicationOfEmail(String email);
```
Comment (ArmineZak) [inline src/main/java/com/epam/edumanagementsystem/util/service/UserService.java:17]: For boolean methods it is prefered to check names in if/else context. In this case if (emailExists()).....

File: src/main/java/com/epam/edumanagementsystem/util/service/impl/UserServiceImpl.java [hunk 2]
```diff
@@ -40,4 +40,9 @@ public List<User> findAll() {
     public User findByEmail(String email) {
         return userRepository.findByEmail(email).get();
     }
+
```
Comment (ArmineZak) [inline src/main/java/com/epam/edumanagementsystem/util/service/impl/UserServiceImpl.java:47]: On line 22, please change the variable name byId to user.

File: src/main/java/com/epam/edumanagementsystem/student/rest/service/StudentService.java [hunk 3]
```diff
@@ -2,26 +2,20 @@
...
-    StudentDto update(StudentDto studentDto);
+    Student create(StudentDto studentDto, UserService userService);
 
     Student update(Student student);
 
     List<StudentDto> findAll();
 
-    StudentDto getById(Long id);
-
     Student findByUserId(Long id);
```
Comment (schoolmanagementproject2022) [inline src/main/java/com/epam/edumanagementsystem/student/rest/service/StudentService.java:19]: We should return DTO but not entity

File: src/main/java/com/epam/edumanagementsystem/student/rest/service/StudentService.java [hunk 4]
```diff
@@ -2,26 +2,20 @@
...
 public interface StudentService {
 
     Student create(StudentDto studentDto);
 
-    String delete(StudentDto studentDto);
-
-    StudentDto updateField(StudentDto studentDto);
-
-    StudentDto update(StudentDto studentDto);
+    Student create(StudentDto studentDto, UserService userService);
```
Comment (schoolmanagementproject2022) [inline src/main/java/com/epam/edumanagementsystem/student/rest/service/StudentService.java:13]: We should return DTO but not entity, and in method we should pass entity but not DTO

File: src/main/java/com/epam/edumanagementsystem/student/rest/service/StudentService.java [hunk 5]
```diff
@@ -2,26 +2,20 @@
...
 import com.epam.edumanagementsystem.student.model.dto.StudentDto;
 import com.epam.edumanagementsystem.student.model.entity.Student;
+import com.epam.edumanagementsystem.util.service.UserService;
 
 import java.util.List;
-import java.util.Optional;
 
 public interface StudentService {
 
     Student create(StudentDto studentDto);
```
Comment (schoolmanagementproject2022) [inline src/main/java/com/epam/edumanagementsystem/student/rest/service/StudentService.java:11]: We should return DTO but not entity, and in method we should pass entity but not DTO

**Review by schoolmanagementproject2022 — APPROVED** [review]

**Pull request closed without merging**